content/200-orm/200-prisma-client/100-queries/058-transactions.mdx

```diff
@@ -311,6 +317,18 @@ await prisma.$transaction(
 )
 ```
 
+You can also set these globally on the constructor-level:
+
+```ts
+const prisma = new PrismaClient({
+  transactionOptions: {
+    isolationLevel: Prisma.TransactionIsolationLevel.Serializable,
+    maxWait: 5000, // default: 2000
+    timeout: 10000, // default: 5000
+  },
+})
+```
+
 <Admonition type="warning">
```

Comment:
This warning now feels a bit out of plce under "Transaction Option". Does it need to be moved up maybe?

Reply:
Good catch! Moved it to the first section before the example 👍